src/discovery/azure/Akka.Discovery.Azure/Actors/HeartbeatActor.cs

```diff
@@ -74,7 +74,7 @@ protected override void OnReceive(object message)
                     if(_log.IsDebugEnabled)
                         _log.Debug("Updating cluster member entry TTL");
 
-                    ExecuteUpdateOpWithRetry().PipeTo(Self);
+                    ExecuteUpdateOpWithRetry().PipeTo(Self, success: () => Status.Success.Instance);
```

Comment:
What happens on failure?

Reply:
`PipeTo` always emits a `Status.Failure` on exceptions, unless you override it with a failure callback. The problem is with `PipeTo` that does not return anything (`async Task`, not `async Task<T>`), they're more akin to starting a detached `Task` that returns a `Status.Failure` when an exception was thrown

Reply:
That would do it. Got it, makes sense.